src/objects/zcl_abapgit_object_clas.clas.abap

```diff
@@ -37,15 +38,20 @@ CLASS zcl_abapgit_object_clas DEFINITION PUBLIC INHERITING FROM zcl_abapgit_obje
         RAISING   zcx_abapgit_exception.
 ENDCLASS.
 
+
+
 CLASS zcl_abapgit_object_clas IMPLEMENTATION.
 
+
   METHOD constructor.
     super->constructor( is_item     = is_item
                         iv_language = iv_language ).
 
     CREATE OBJECT mi_object_oriented_object_fct TYPE zcl_abapgit_oo_class_new.
+    mv_activation_placeholder_step = abap_true.
```

Comment:
The bad thing about this is that it introduces state to the deserializer. Another option would be to have a completely different deserializer only for the class "name"